src/components/ConversationSettings/LinkShareSettings.vue

```diff
@@ -25,26 +25,25 @@
 			{{ t('spreed', 'Guest access') }}
 		</h4>
 
-		<NcCheckboxRadioSwitch :checked="isSharedPublicly"
-			:disabled="isSaving"
-			type="switch"
-			aria-describedby="link_share_settings_hint"
-			@update:checked="toggleGuests">
-			{{ t('spreed', 'Allow guests to join this conversation via link') }}
-		</NcCheckboxRadioSwitch>
-
-		<NcCheckboxRadioSwitch v-show="isSharedPublicly"
-			:checked="conversation.hasPassword"
-			:disabled="isSaving"
-			type="switch"
-			aria-describedby="link_share_settings_password_hint"
-			@update:checked="togglePassword">
-			{{ t('spreed', 'Password protection') }}
-		</NcCheckboxRadioSwitch>
-
-		<template v-if="showPasswordField">
-			<form class="password-form"
-				@submit.prevent="handleSetNewPassword">
+		<template v-if="canFullModerate">
+			<NcCheckboxRadioSwitch :checked="isSharedPublicly"
+				:disabled="isSaving"
+				type="switch"
+				aria-describedby="link_share_settings_hint"
+				@update:checked="toggleGuests">
+				{{ t('spreed', 'Allow guests to join this conversation via link') }}
+			</NcCheckboxRadioSwitch>
+
+			<NcCheckboxRadioSwitch v-show="isSharedPublicly"
+				:checked="conversation.hasPassword"
```

Comment:
`hasPassword` is not updated
![image](https://github.com/nextcloud/spreed/assets/84044328/7815ff9c-c3e6-4091-8a33-bfecebfe8841)

Reply:
[toggle-password.webm](https://github.com/nextcloud/spreed/assets/93392545/b561d3ca-5a3c-4b78-b4d0-c5e5b447e141)